TwitchPlaysAssembly/Src/Commands/GameCommands.cs

```diff
@@ -579,6 +579,32 @@ public static void CallSetCommand(string user, [Group(1)] int minimum)
 	[Command(@"call *count")]
 	public static void CallCountCommand() => IRCConnection.SendMessageFormat("{0} out of {1} calls needed.", TwitchGame.Instance.CallingPlayers.Count, TwitchGame.Instance.callsNeeded);
 
+	[Command(@"uncall")]
+	public static void RemoveQueuedPlayer(string user)
+	{
+		if (!(TwitchGame.Instance.CallingPlayers.Contains(user)))
+		{
+			IRCConnection.SendMessage($"@{0}, you haven't called yet!", user);
```

Comment:
You used both an interpolated string (`$"{<code>}"`) and a formatted string (`"{<number>}"`) every time you used SendMessage. Since the interpolated string gets turned into a valid formatted string it doesn't cause any errors but you should just use one or the other.